lib/bolt/core/src/dep/terraform/install_scripts/components.rs

```diff
@@ -381,14 +381,14 @@ pub async fn traffic_server(ctx: &ProjectContext, server: &Server) -> Result<Str
 		.map(|(k, v)| format!("cat << 'EOF' > /etc/trafficserver/{k}\n{v}\nEOF\n"))
 		.collect::<Vec<_>>();
 
-	// Update default storage config size to be entire filesystem size
+	// Update default storage config size to be entire filesystem size minus 2GB
 	config_scripts.push(
 		indoc!(
-			"
-		df -h / |
-		awk 'NR==2 {print $2}' |
-		xargs -I {} sed -i 's/64G/{}/' /etc/trafficserver/storage.config
-		"
+			r#"
+			df -h / |
```

Comment:
this was wrong in the first place. we should be using the total file system size, not df since df returns inconsistent responses that vary based on everything else on the system.

gpt is saying this is wrong too, can you double check that the output of this script is accurate? maybe drop a `cat` somewhere in the script with the new storage.config in order to validate it?

![image.png](https://graphite-user-uploaded-assets-prod.s3.amazonaws.com/BIF3m0zL7mcrHUeMWh4D/faf0f08d-1a08-4720-95d5-d97263a67d15.png)

Reply:
The script is correct, and its exactly what chatgpt printed for you after step 4. I'm not sure where it got step 2 from, thats from the old script that i modified. Did you paste in the entire diff?

![image.png](https://graphite-user-uploaded-assets-prod.s3.amazonaws.com/WoaiNOUjGiXic7rsR2cI/a58e8da8-5123-4250-991f-e58aec9ddd2d.png)

What do i used besides `df`? `lsblk`?

Reply:
Oh whoops, `df` is totally fine.